### PR TITLE
Removed unnecessary &nbsp; entities from the admin.

### PR DESCRIPTION
**Pull request by jdufresne**
Normal spaces are fine when used as a single spaced separation. For
larger spacing, CSS is used.

### CODE REVIEW

**Comment by elky** [comment]
I totally like it. `&nbsp;`s were a headache for me when I worked on responsive version and according to `git blame` they went from the early beginning. Bye bye!

File: django/contrib/admin/templates/admin/login.html [hunk 1]
```diff
@@ -58,7 +58,7 @@
   </div>
   {% endif %}
   <div class="submit-row">
-    <label>&nbsp;</label><input type="submit" value="{% translate 'Log in' %}">
+    <input type="submit" value="{% translate 'Log in' %}">
```
Comment (carltongibson) [inline django/contrib/admin/templates/admin/login.html:61]: This affects the layout by a small number of pixels. **But** the button isn't centered perfectly anyway... — so it's OK I think. 

(Might be a place where flexbox would be a better choice now, than a hard-coded padding value.)

**Comment by carltongibson** [comment]
buildbot, test on selenium.

File: django/contrib/admin/templates/admin/login.html (hunk 1 above)
Comment (jdufresne) [reply django/contrib/admin/templates/admin/login.html:61]: Do you know if there is any reason this element isn't styled as `align: text-center`? Is that a RTL issue? If there is no issue, I can look at making that change so it is centered.

Comment (carltongibson) [reply django/contrib/admin/templates/admin/login.html:61]: I don't know, no. It had always appeared centred, but when I got the ruler out... :)

Comment (jdufresne) [reply django/contrib/admin/templates/admin/login.html:61]: I went down a bit of a rabbit hole cleaning up and simplifying the admin CSS.

The fix to center the button is in #12847.

**Pull request merged**